packages/app/src/DBSearchPage.tsx

```diff
@@ -132,26 +132,33 @@ function SearchTotalCount({
     granularity,
     limit: { limit: 100000 },
   };
-  const {
-    data: totalCountData,
-    isLoading,
-    isError,
-  } = useQueriedChartConfig(queriedConfig, {
-    queryKey: [queryKeyPrefix, queriedConfig],
-    staleTime: 1000 * 60 * 5,
-    refetchOnWindowFocus: false,
-  });
+  const { data: totalCountData, isError } = useQueriedChartConfig(
+    queriedConfig,
+    {
+      queryKey: [queryKeyPrefix, queriedConfig],
+      staleTime: 1000 * 60 * 5,
+      refetchOnWindowFocus: false,
+    },
+  );
 
-  const totalCount = useMemo(() => {
-    return totalCountData?.data?.reduce(
-      (p: number, v: any) => p + Number.parseInt(v['count()']),
-      0,
+  const [totalCount, setTotalCount] = useState<number | null>(null);
```

Comment:
just for future reference this is built in to react query with: https://tanstack.com/query/v5/docs/framework/react/guides/paginated-queries#better-paginated-queries-with-placeholderdata

you should be able to use `placeholderData` option, I'm working on a small refactor of this component so I'll put in this change in a PR i'm working on